stdlib/public/core/Zip.swift

```diff
@@ -136,13 +147,140 @@ extension Zip2Sequence.Iterator: IteratorProtocol {
 }
 
 extension Zip2Sequence: Sequence {
-  public typealias Element = (Sequence1.Element, Sequence2.Element)
```

Comment:
This is intentionally explicit, please don't remove it.

Reply:
👍